tmt/steps/provision/mrack.py

```diff
@@ -1308,11 +1405,31 @@ def is_ready(self) -> bool:
         except mrack.errors.MrackError:
             return False
 
-    def _create(self, tmt_name: str) -> None:
+    @property
+    def _bootc_registry_credentials(self) -> dict[str, Any]:
+        """Create authentication dictionary with proper credential handling"""
+        if not self.data.registry_secret:
+            return {}
+        assert self.data.bootc_image_url
+        base_repo = _get_registry_from_url(self.data.bootc_image_url)
+        # Support multiple registries
+        return {"auths": {base_repo: {"auth": self.data.registry_secret}}}
+
+    def start(self) -> None:
         """
-        Create beaker job xml request and submit it to Beaker hub
+        Start the guest
+
+        Get a new guest instance running. This should include preparing
+        any configuration necessary to get it started. Called after
+        load() is completed so all guest data should be available.
         """
 
+        if self.job_id and self.primary_address:
+            return
+        """
+        Create beaker job xml request and submit it to Beaker hub
+        """
```

Comment:
This should be either a comment, or gone. It no longer works as a docstring, now it's just a multiline string not assigned to any variable.